charts/lm-container/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: lm-container
 description: A Helm chart for Logicmonitor's Kubernetes monitoring solutions
 type: application
-version: 4.3.0-rt01
+version: 4.4.0-rc01
```

Comment:
4.4.0-rc01 is now published, please increase the version. say, 4.4.0-rc02

Reply:
Done.